recipes/lcaclassifier/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: lcaclassifier
+  version: 3.0.7
+
+source:
+  url: https://github.com/lanzen/CREST/releases/download/3.0.7/LCAClassifierV3.0.7.tar.gz
+  md5: ca072806b9251f280978c6a3ba520395
+  patches:
+    - lcaclassifier.patch
+
+build:
+  skip: True  # [py3k]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biom-format
+    - biopython
+
+  run:
+    - python
```

Comment:
Acc. to `setup.py`, `setuptools` is also a run requirement.
But I can't see why, so you can either
1. add a patch to remove `setuptools` from `install_requires` in `setup.py` (preferred), or
2. add add `setuptools` to the run requirements